releases/release-1.22/release-notes/maps/pr-100694-map.yaml

```diff
@@ -0,0 +1,3 @@
+pr: 100694
+releasenote:
+  text: 'Fix: not deleting existing publich IP when service is deleted in Azure'
```

Comment:
@pmmalinov01 Should there be single quotes around the entire text body?

Reply:
Single quotes need to be used to prevent the extra `:` to be detected as YAML key.